src/loader.rs

```diff
@@ -552,12 +565,12 @@ impl<'b, 'w> Loader<'b, 'w> {
             _ => format!("FbxMaterial{}", material_obj.object_id().raw()),
         };
         if let Some(handle) = self.scene.materials.get(&label) {
-            trace!("Already encountered material: {label}, skipping");
+            warn!("Already encountered material: {label}, skipping");
 
             return Ok(handle.clone_weak());
         }
 
-        trace!("Loading material: {label}");
+        info!("Loading FBX material: {label}");
```

Comment:
```suggestion
        debug!("Loading FBX material: {label}");
```
I'd rather not enable it on `info`, this seems too verbose.